[IT] Review multiple articles (#13569)
ppy/osu-wiki: wiki/Reporting_bad_behaviour/Abuse/it.md

Nulla di ciò che invii a [abuse@ppy.sh](mailto:abuse@ppy.sh) sarà mai condiviso con terzi.

## Qualcuno ha fatto avance sessuali o osservazioni inappropriate su di me!

Scorri alle sezioni qui sotto per scoprire cosa fare, a seconda di dove è accaduto tutto.

### Nella chat di gioco

Blocca immediatamente l'utente cliccando sul suo username col tasto destro e selezionando `Ignora Utente`.

Dovresti anche segnalarlo immediatamente cliccando il pulsante `Segnala Utente` e compilando i campi.

Puoi segnalare l'utente anche scrivendo `!report <persona> <problema>`, dove `<persona>` è il nome utente della persona che sta causando problemi, e `<problema>` è una breve descrizione del problema. Se ci sono spazi nel nome utente, rimpiazzali con trattini bassi (`_`).

Se ritieni che le osservazioni o le avance fossero altamente inappropriate o sei un utente minorenne, per favore invia un'email a [abuse@ppy.sh](mailto:abuse@ppy.sh) includendo quante più informazioni possibili.

### In un post pubblico sui forum di osu!

[Segnala il post](/wiki/Reporting_bad_behaviour#forum) al nostro [Team di Moderazione Globale](/wiki/People/Global_Moderation_Team) il prima possibile.

> Per fare ciò, clicca sul pulsante `Segnala post` nell'angolo in alto a destra del post del forum.

Reviewer: ```suggestion
Per farlo, clicca sul pulsante `Segnala post` nell'angolo in alto a destra del post del forum.
```

> In alternativa, clicca sul nome utente di un qualsiasi membro del GMT nella lista su quella pagina. Questo ti porterà al suo profilo. Se non sei sicuro di chi contattare, usa [invece questo link](https://osu.ppy.sh/users/5999631).

Reviewer: ```suggestion
In alternativa, clicca sul nome utente di un qualsiasi membro del GMT nella lista su quella pagina. Questo ti porterà al suo profilo. Se non sei sicuro di chi contattare, usa [questo link](https://osu.ppy.sh/users/5999631).
```

Clicca il pulsante con una busta appena sotto la bandiera del paese dell'utente. Questo ti porterà alla schermata della chat.

Incolla il link del post del forum con il contenuto problematico e spiega brevemente cosa sta succedendo al GMT. Rimuoveranno il post per te appena gli sarà possibile.

Se il post non viene rimosso entro un periodo di tempo ragionevole, per favore, invia un'email a [abuse@ppy.sh](mailto:abuse@ppy.sh) includendo quante più informazioni possibili.

### Su un server Discord pubblico relativo a osu!

> Blocca immediatamente l'utente cliccando col tasto destro sul suo nome utente e selezionando `Blocca` dal menu a tendina che appare.

Reviewer: ```suggestion
Blocca immediatamente l'utente cliccando col tasto destro sul suo nome utente e selezionando `Blocca` dal menu contestuale.
```
